CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+This project is based on
+
+- TypeScript
+- Rust
+- C# .NET
+
+By that mean, some dependencies are required to work with.
+Each of them is defined in the project folder.
+
+## Projects
+
+- [vrc-get CLI](vrc-get\README.md)
+- [vrc-get LiteDB](vrc-get-litedb\README.md)
+- [vrc-get GUI](vrc-get-gui\README.md)
+- [vrc-get VPM](vrc-get-vpm\README.md)
+
+## Configuration requirements
+
+You can work on any OS system but this repository uses
+
+- Git Submodules
+- Symbolic Links
+
+For Windows machines, you need to setup so your current user can create symbolic links. Refer to this documentation page <https://github.com/git-for-windows/git/wiki/Symbolic-Links>
+
+To setup your project, use the following command to clone with submodules and symbolic links enabled.
+
+`git clone --recurse-submodules -c core.symlinks=true https://github.com/vrc-get/vrc-get`
```

Comment:
`-c core.symlinks=true` can be not required on some platform so I think it should be omitted.

In addition, this is complete code so I think it's better to use code block

````suggestion
```bash
git clone --recurse-submodules -c core.symlinks=true https://github.com/vrc-get/vrc-get
```
````

Reply:
Should be done Now